perfkitbenchmarker/object_storage_multistream_analysis.py

```diff
@@ -98,8 +100,15 @@ def GetStreamActiveIntervals(start_times, durations, stream_ids):
   interval_start = stream_start_times.max()
   interval_end = stream_end_times.min()
 
-  return (Interval(first_start, end=last_end),
-          Interval(interval_start, end=interval_end))
+  logging.info('Stream ends after stream starts: %s',
+               (stream_end_times > stream_start_times).all())
+  if interval_end <= interval_start:
+    raise Exception('No interval when all streams were active.')
```

Comment:
Be more specific on Exception raised. e.g. define something like ObjectStorageServiceException